client/src/components/Header/Header.js

```diff
@@ -14,7 +14,13 @@ class Header extends Component {
   }
 
   renderHeaderButtons () {
-    if (this.props.authenticated) {
+    if (this.props.authenticated && this.props.user.usertype === 'contact') {
```

Comment:
This could possibly be refactored to something like
```
if (this.props.authenticated) {
  const authButtons = [
    <Link key="logout" to='/' onClick={this.props.logout} className={styles.button}>LOG OUT</Link>,
    <Link key="profile" to='/profile' className={styles.button}>PROFILE</Link>
  ]

  if (this.props.user.usertype === 'contact') {
    return [
      <Link key="create-project" to='/create-project' className={styles.button}>CREATE PROJECT</Link>,
      ...authButtons
    ]
  } else {
    return authButtons
  }
} else { ... }
```

This way when we add more buttons we won't need to repeat code anywhere